generators/kubernetes/templates/kustomize/patch/istio-namespace.yml.ejs

```diff
@@ -0,0 +1,3 @@
+- op: replace
+  path: "/metadata/namespace"
+  value: "istio-system"
```

Comment:
is there a reason to patch istio-system namespace here? what is this used for?

Reply:
This is to expose resources like zipkin, kiali etc. running in istio-system namespace